DiceKit/MultiplicationExpressionResult.swift

```diff
@@ -39,3 +39,27 @@ public struct MultiplicationExpressionResult<MultiplierResult: protocol<Expressi
 public func == <M, R>(lhs: MultiplicationExpressionResult<M,R>, rhs: MultiplicationExpressionResult<M,R>) -> Bool {
     return lhs.multiplierResult == rhs.multiplierResult && lhs.multiplicandResults == rhs.multiplicandResults
 }
+
+// MARK: - CustomDebugStringConvertible
+
+extension MultiplicationExpressionResult: CustomDebugStringConvertible {
+    
+    public var debugDescription: String {
+        get {
+            return "\(String(reflecting: multiplierResult)) * \(String(reflecting: multiplicandResults))"
+        }
+    }
+    
+}
+
+// MARK: - CustomStringConvertible
+
+extension MultiplicationExpressionResult: CustomStringConvertible {
+    
+    public var description: String {
+        get {
+            return "\(multiplierResult) * \(multiplicandResults)"
```

Comment:
So, this one is incorrect. The `multiplierResult` was used to generate all of the `multiplicandResults`. I believe that here you just show all of them being added together: `(5 +  3+ 1 + 5 + 2)`